en.js

```diff
@@ -69,12 +69,12 @@ module.exports = {
                 "s"
             ]
         },
-        command_maintenance_text_3: "🏳️** • Traitement des données »** Server Analytics est actuellement en processus **quotidien** de traitement des données, pour éviter une surcharge du réseau et veiller à la continuité de nos services, reviens dans ` ${mtnc.duration} `"
+        command_maintenance_text_3: "🏳️** • Data processing »** Server Analytics is currently in the data processing **daily** process, to avoid a network overload and ensure the continuity of our services, come back in ` ${mtnc.duration} `"
```

Comment:
La première partie de la phrase sonne étrange (dans mon opinion) et il y a une répétition trop marquée du mot "process"/"processing", j'ai fait une tentative de reformulation, peut-être est-elle un chouilla plus adaptée ?
`Server Analytics is currently processing all the statistics that have been recorded **today**`